src/View/ViewableData.php

```diff
@@ -261,6 +268,14 @@ public function hasDynamicData(string $field): bool
         return array_key_exists($field, $this->dynamicData);
     }
 
+    /**
+     * Whether the model requires sudo mode to be active in order to be modified in admin
+     */
+    public function getRequireSudoMode(): bool
+    {
+        return $this->config()->get('require_sudo_mode');
+    }
```

Comment:
Why do we have a method that just returns a config value? Shouldn't we just check the config directly instead? Or is the idea that people can override this method to do some complex conditional logic in here if they want to?

Reply:
LeftAndMain has `if (is_a($record, DataObject::class) && $record->getRequireSudoMode()) {` - I think it's nicer to call a method in that context rather than reading another classes config

I have just made an update to change this to use static::config() instead